lib/msf/core/exploit.rb

```diff
@@ -81,28 +81,28 @@ class CheckCode < Struct.new(:code, :message, :reason)
     # of the codebase.
 
     class << self
-      def Unknown(reason = nil)
-        self.new('unknown', reason)
+      def Unknown(reason = nil, **kwargs)
+        self.new('unknown', reason, **kwargs)
```

Comment:
Should these still pass `**kwargs` or should they switch to take an optional `details`?
```suggestion
      def Unknown(reason = nil, details = {})
        self.new('unknown', reason, details = {})
```